packages/api/models/granules.js

```diff
@@ -132,6 +132,37 @@ class Granule extends Manager {
     return this.update({ granuleId: granule.granuleId }, { published: false }, ['cmrLink']);
   }
 
+  /**
+   * With the params for moving a granule, return the files that already exist at
+   * the move location
+   *
+   * @param {Object} granule - the granule object
+   * @param {Array<{regex: string, bucket: string, filepath: string}>} destinations
+   * - list of destinations specified
+   *    regex - regex for matching filepath of file to new destination
+   *    bucket - aws bucket of the destination
+   *    filepath - file path/directory on the bucket for the destination
+   * @returns {Promise<Array<Object>>} - promise that resolves to a list of files
+   * that already exist at the destination that they would be written to if they
+   * were to be moved via the move granules call
+   */
+  async getFilesExistingAtLocation(granule, destinations) {
+    const moveFileParams = generateMoveFileParams(granule.files, destinations);
```

Comment:
Missing this function `generateMoveFileParams` and `s3Utils` with this reversion of `getFilesExistingAtLocation` here. I guess `eslint` errors got overlooked since no CI run on this branch. :)